test/crawler/keyword/keywords_test.exs

```diff
@@ -106,6 +106,27 @@ defmodule Crawler.Keyword.KeywordsTest do
     end
   end
 
+  describe "get_keyword_by_user_id_and_id/2" do
+    test "given a valid keyword id, returns stored keyword" do
+      %{id: user_id} = insert(:user)
+      keyword = insert(:keyword, user_id: user_id, name: "keyword")
+
+      assert stored_keyword = Keywords.get_keyword_by_user_id_and_id(user_id, keyword.id)
+      assert stored_keyword.id == keyword.id
+      assert stored_keyword.name == "keyword"
+      assert stored_keyword.user_id == user_id
+    end
+
+    test "given a keyword belongs to another user, returns nil" do
```

Comment:
let's add the test for `given a non-existing keyword id, returns nil`

Reply:
Added in [3dfd2cd](https://github.com/vnntsu/elixir-ic/pull/47/commits/3dfd2cd51f959f48690b21d0820b5b2345d23323)